docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -101,6 +101,7 @@ Some directories in the container are mounted, which means that their contents a
 
 - ``./dags`` - you can put your DAG files here.
 - ``./logs`` - contains logs from task execution and scheduler.
+- ``./config`` - you can add custom log parser or add ``airflow_local_settings.py`` to configure cluster policy.
```

Comment:
Hm. But I believe config is not mounted :

```
  volumes:
    - ${AIRFLOW_PROJ_DIR:-.}/dags:/opt/airflow/dags
    - ${AIRFLOW_PROJ_DIR:-.}/logs:/opt/airflow/logs
    - ${AIRFLOW_PROJ_DIR:-.}/plugins:/opt/airflow/plugins
```

Reply:
oops i thought its missed out in documentation part alone. could i add that as part of the docker-compose.yml file too @potiuk ?

Reply:
@potiuk could you let me know if i can add as part of docker-compose.yml? bringing this up as a gentle reminder

Reply:
@potiuk I think it mounted here https://github.com/apache/airflow/blob/90ee7639478bc8534164c4bbb05a1ccfb04272dd/docs/apache-airflow/howto/docker-compose/docker-compose.yaml#L75

or do we need to mount it at some other places too?

Reply:
config?

Reply:
> @potiuk could you let me know if i can add as part of docker-compose.yml? bringing this up as a gentle reminder

Yes. you can.

Reply:
> config? @pankajastro ?

yeah it missing 🤦‍♂️